server/routerlicious/packages/lambdas/src/utils/helpers.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Converts the given number array into an array of ranges
+ * Example: [1, 2, 3, 4, 5, 6] to [[1, 6]]
+ * [1, 2, 3, 5, 6] to [[1,3],[5,6]]
+ */
+export function convertNumberArrayToRanges(numberArray: number[]): number[][] {
+    const ranges: number[][] = [];
+        if (numberArray.length === 0) {
+            return ranges;
+        }
+        let begin: number = numberArray[0];
+        let end: number = numberArray[0];
+        let count = 0;
+        for (let i = 1; i < numberArray.length; i++) {
+            const elem = numberArray[i];
+            count++;
+            if (elem - begin !== count) {
+                ranges.push([begin, end]);
+                count = 0;
+                begin = elem;
+                end = elem;
+            } else {
+                end = elem;
+            }
+        }
+
+        if (begin !== undefined) {
+            ranges.push([begin, end]);
+        }
+
+        return ranges;
+}
```

Comment:
@znewton  I understand the lambdas is independent. But I feel this `convertNumberArrayToRanges` is reusable for others. By looking at the lambdas dependencies:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/3719246/205079032-0a35628b-6e65-43d8-8364-f1ee830b4efd.png">


Is it OK to move this function to `common-utils` or `server-service-core`?

Reply:
Hey @zhangxin511, thanks for pointing out that you want to re-use this utility from FRS.. @znewton is there any guideline on where the utils live that are used by both OSS and FRS?
Also @zhangxin511 is it ok if I follow up on this in the next PR so that scriptorium logging fixes are unblocked, thanks!